spec/units-of-measure.md

```diff
@@ -299,66 +265,77 @@ The F# core library defines the following types:
 type float<[<Measure>] 'U>
 type float32<[<Measure>] 'U>
 type decimal<[<Measure>] 'U>
-type int<[<Measure>] 'U>
 type sbyte<[<Measure>] 'U>
 type int16<[<Measure>] 'U>
+type int<[<Measure>] 'U>
 type int64<[<Measure>] 'U>
+type nativeint<[<Measure>] 'U>
+type uint<[<Measure>] 'U>
+type byte<[<Measure>] 'U>
+type uint16<[<Measure>] 'U>
+type uint64<[<Measure>] 'U>
+type unativeint<[<Measure>] 'U>
 ```
 
-> Note: These definitions are called measure-annotated base types and are marked with
-the `MeasureAnnotatedAbbreviation` attribute in the implementation of the library. The
-`MeasureAnnotatedAbbreviation` attribute is not for use in user code and in future
-revisions of the language may result in a warning or error.
+> Note: These definitions are called measure-annotated base types and are marked with the `MeasureAnnotatedAbbreviation` attribute in the implementation of the library. The `MeasureAnnotatedAbbreviation` attribute is not for use in user code and in future revisions of the language may result in a warning or error.
```

Comment:
Due to its use in UMX https://github.com/fsprojects/FSharp.UMX/blob/master/src/FSharp.UMX.fs , this "threat" is unlikely to happen - we would treat such removal as a change breaking normal real-world code by now.


Personally, the remark could be deleted in this revision of the spec.